Sources/Protocols/Keyboardable.swift

```diff
@@ -9,26 +9,36 @@
 import Foundation
 import UIKit
 
-public protocol Keyboardable {
+protocol Keyboardable: class {
+    var keyboardObserver: [Any] { get set }
     func keyboardChanges(height: CGFloat)
 }
 
-public extension Keyboardable {
+extension Keyboardable { //where Self: UIViewController {
```

Comment:
Again the case with public accessors. As for the commented out `where`, I think it's not that important to have it there so you can remove it.